crates/glaredb/src/commands.rs

```diff
@@ -33,6 +42,9 @@ pub enum Commands {
     /// Starts an instance of the Metastore.
     #[clap(hide = true)]
     Metastore(MetastoreArgs),
+    /// Runs SQL Logic Tests
+    #[clap(hide = true)]
```

Comment:
can we alias this to `slt`

```suggestion
    #[clap(hide = true, alias="slt")]
```